content/blog/2021-09-30-september-21-community-gems.md

```diff
@@ -0,0 +1,125 @@
+without needing to duplicate it on each machine.
+
+Thanks for the question @fchpriani!
+
+### [If we change the remote we are using in our workspace, does that effect where DVC pulls and pushes data to for all historical commits?](https://discord.com/channels/485586884165107732/563406153334128681/882951655979622400)
+
+Thanks for bringing this up @mattlbeck!
+
+Right now DVC just uses whichever remote is configured in a respective commit
+that you've checked out.
+
+To clarify things a bit more, if you run `dvc push/pull` in a workspace with new
+a remote, that new remote will be used for `-a`, `-T`, and `--all-commits`.
+
+### [Is there a command to execute only a few specific stages in a DVC pipeline?](https://discord.com/channels/485586884165107732/485596304961962003/888054401640562698)
+
+You can freeze the stages that you do not want to be executed.
+
+`dvc freeze` and `dvc unfreeze` help you do this. Or you can use
+`dvc repro --glob pattern*` together with `-s` to match the stages you want to
+run.
+
+Thanks for the question @LucZ!
+
+### [When running queued experiments, is it expected for DVC to run `dvc checkout` for each experiment?](https://discord.com/channels/485586884165107732/485596304961962003/883144885417431081)
+
+This brings up a good point, so thanks @dmh!
+
+If you usually run experiments with `dvc repro`, you'll notice that it doesn't
+checkout any files. That's because the experiment is running in the current
+workspace.
+
+When you use `dvc exp run --queue` or `dvc exp run --run-all`, it runs each
+experiment in its own separate temp workspace, so files have to be checked out
+into those workspaces. Check out the notes in
+[this reference doc on queueing and parallel execution](https://dvc.org/doc/command-reference/exp/run#queueing-and-parallel-execution)
+for more details.
+
+### [When working with a data registry, is it possible to pull a specific project folder, modify it, then push Git changes and `dvc push` to the remote storage without pulling data from all the directories?](https://discord.com/channels/485586884165107732/485596304961962003/887427010044002345)
+
+This is definitely possible. You can use a Git sub-repo and a DVC sub-repo to do
+this. Use `git init` and `dvc init` in the project folders and then you can pull
+them down, modify, commit and push commit back.
+
+This is the most seure way to handle this use case, but you have another option.
+You can `dvc pull` from the sub-directory, then make your changes in the
+sub-directory, and `dvc add` the changes. Then you can do a `git commit` and
+`dvc push` to manage those changes.
+
+Really good question @ross.tsenov!
+
+### [Is it possible to auto-generate reports with metrics and plots when the data is stored in an AWS bucket instead of GitHub?](https://discord.com/channels/485586884165107732/728693131557732403/877072469188575262)
+
+Thanks for asking @Masmoudi!
+
+If the metrics and plots are not being stored in S3, then you may be able to
```

Comment:
This answer reads a little weird to me. The initial question isn't really related to the issue the user ultimately had. 

They were trying to run dvc within a cml job, but the default cml action doesn't install dvc. See the note at the end of this section: https://cml.dev/doc/start/github#the-cml-github-action.

You could probably get rid of this whole paragraph and start the next paragraph with "When you need to retrieve DVC data in CML."